test/xpu_api/numerics/numeric.ops/reduce/xpu_reduce.pass.cpp

```diff
@@ -0,0 +1,108 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <numeric>
+// UNSUPPORTED: c++03, c++11, c++14
+// UNSUPPORTED: clang-8
+// UNSUPPORTED: gcc-9
+
+// Became constexpr in C++20
+// template<class InputIterator>
+//     typename iterator_traits<InputIterator>::value_type
+//     reduce(InputIterator first, InputIterator last);
+
+#include <oneapi/dpl/numeric>
+#include <cassert>
+#include <CL/sycl.hpp>
+
+#include "support/test_iterators.h"
+
+constexpr sycl::access::mode sycl_read = sycl::access::mode::read;
```

Comment:
You use those constants in one place. If you like to have shorter name in usage place, why cannot you make these variables definition in place where you use it? I just don't see the reason for them to be global.